test-apps/ui-test-app/src/frontend/index.tsx

```diff
@@ -376,7 +376,7 @@ export class SampleAppIModelApp {
         redirectUri: "http://localhost:3000/esri-oauth2-callback",
         clientIds: {
           arcgisOnlineClientId: SampleAppIModelApp?.testAppConfiguration?.arcGisOnlineClientId,
-          enterpriseClientIds,
+          enterpriseClientIds: [{serviceBaseUrl: "", clientId: "Bentley_TestApp"}],

```

Comment:
Are you intentionally overwriting the `enterpriseClientIds` you computed above?

Reply:
the default value will not override other `enterpriseClientIds ` values provided, it just a default value if no match is found.